lib/stroke.py

```diff
@@ -85,7 +86,7 @@ def render(self, surface):
         states = np.fromstring(self.brush_state, dtype='float32')
         b.set_states_from_array(states)
 
-        version, data = self.stroke_data[0], self.stroke_data[1:]
+        version, data = self.stroke_data[0:1], self.stroke_data[1:]
```

Comment:
not sure what this "version" variable is needed for anyway, but like this it seems to work for both py2 / py3